Sources/MapboxNavigation/NavigationCamera.swift

```diff
@@ -125,10 +125,15 @@ public class NavigationCamera: NSObject, ViewportDataSourceDelegate {
      Call to this method executes a transition to `NavigationCameraState.following` state.
      When started, state will first change to `NavigationCameraState.transitionToFollowing` and then
      to the final `NavigationCameraState.following` when ended.
+     
+     - parameter completion: Completion handler, which is called whenever transition ends or doesn't
+     occur at all (e.g. in case if already in `NavigationCameraState.transitionToFollowing` or
+     `NavigationCameraState.following` state).
      */
-    public func follow() {
+    public func follow(_ completion: (() -> Void)? = nil) {
```

Comment:
Decided to add this optional completion handler to public API to be able to test behavior more easily.

Reply:
There is no need to do this here as it's public API anyway.